app-containers/podman/metadata.xml

```diff
@@ -28,7 +28,10 @@
 			necessary kernel flags.
 		</flag>
 		<flag name="cgroup-hybrid">
-			Default to hybrid (legacy) cgroup hierarchy instead of unified (modern).
+			Use legacy (hybrid) cgroups instead of modern (unified) cgroups
+		</flag>
+		<flag name="docker">
+			Install wrapper which lets use podman for command `docker`
```

Comment:
Maybe we should call this `docker-wrapper`, since USE=docker might imply support for the real docker in the context of other ebuilds.

Reply:
Is it really worth introducing new use flag in tree just for clarity? I mean we do have description for that. I also see your point though, we can use 'wrapper' use flag as it is already there.